### DISCUSSION TITLE
SSTV正式入驻绳网啦！

### DISCUSSION THREAD

**Discussion by SSTV-10068** [share121/inter-knot#907] [其他]
SSTV正式入驻绳网拉！SSTV主要负责向大家分享各种有趣的趣闻，欢迎大家踊跃投稿
投稿邮件：a1396228851@qq.com
![111](https://github.com/user-attachments/assets/17b1a7b8-396b-4b44-9849-df7b6ed80505)

**Comment by Ropemakers5302** [comment]
真假的，以后都播报啥啊

**Comment by SSTV-10068** [comment]
各种趣闻，欢迎各位投稿
